pwnlib/elf/corefile.py

```diff
@@ -427,7 +427,7 @@ class Corefile(ELF):
 
         Corefiles can also be pulled from remote machines via SSH!
 
-        >>> s = ssh('travis', 'example.pwnme')
+        >>> s = ssh(host='example.pwnme', user='travis', password='demopass')
```

Comment:
The testing infrastructure already installs an ssh key for user `travis` and configures it.
Also, this demonstrates the way to connect over ssh (resembling `travis@example.pwnme`).
```suggestion
        >>> s = ssh('travis', 'example.pwnme')
```

Reply:
As far as i remember (from november 2018, aka 6+ months ago) it didn't work for me without a password for me.

And as the parameter-order is kind of weird (user, host, port, password) - the ssh-command-structure vs the every-other-protocol-structure of going more specific (which host - what port - what user - what's his password) - I'd prefer the explicit param names for clarity (as changing it would break APIs)